src/index.js

```diff
@@ -93,6 +93,8 @@ class GifPlayerContainer extends React.Component {
   toggle () {
     this.setState({
       playing: !this.state.playing
+    }, () => {
+      this.props.onChange && this.props.onChange(this.state.playing)
```

Comment:
A few comments:
1. I prefer the form:
```js
if (typeof this.props.onChange === 'function') {
  functionCall();
}
```
2. I also prefer handling this in `componentDidUpdate` - we can just check if the playing state is different than the last.
3. Semicolon would be great. 😉